webui/src/components/KeyResults.vue

```diff
@@ -77,8 +101,16 @@
                     objective: objective,
                     keyresult: keyresult,
                     krStatement: keyresult.statement,
-                    krPercent: keyresult.percent
+                    krPercent: keyresult.percent,
+                    krDescription: keyresult.description
                 });
+            },
+
+            openChat(keyresultId) {
+                this.showDescription = !this.showDescription;
+
+                // TODO : FIND InputForm html-element WHERE :name == keyresultId
```

Comment:
I think we should not manipulate HTML elements and their styles, but instead introduce the KeyResult (without -s) components, which has InputField, progress, chat icon etc. - in this case where will be no need to find the correct InputField because there are the only one.